### ISSUE TITLE
Old Issue Anew

### ISSUE THREAD

**Issue by MGParisi** [mpdf/mpdf#810]
Just got this issue.  Works on my local but the remote linux machine doesn't accept it.  Very strange.

The error is the same...
https://github.com/mpdf/mpdf/issues/44

Solution Found: 
I added a heading with <divs>.  In Windows Server this works, but in Linux it fails with the error.  When I converted these divs to tables it worked in both linux and windows.

**Comment by finwe** [comment]
Closing as per issue template warning. 

Feel free to open a new issue using the provided issue template with all available issue details, especially a code example to easier reproduce the issue.

**Comment by MGParisi** [comment]
Error: Fatal error: require(): Failed opening required 'somepath/wp-content/plugins/dk-pdf/tmp/ttfontdata/dejavusanscondensedB.mtx.php' (include_path='.:/usr/share/pear/php:/usr/share/php:/usr/share/pear') in /nas/content/live/wadurbin/wp-content/plugins/dk-pdf/vendor/mpdf/mpdf/src/Mpdf.php on line 3828

**Comment by lizbanach** [comment]
I have the same issue. Why is this closed when there is no solution?

**Comment by MGParisi** [comment]
Sorry, the ticket was closed because I didn't use a template.

I may have added another one with the template.

I believe its a problem with that font.   I don't remember the resolution.  I usually update tickets when I do, but this was closed so I never expected a response on it.

**Comment by Klap-in** [comment]
@lizbanach an offtopic and general note:
without enough information it's impossible for anybody to reproduce a situation. Therefore we provide a template that you fill if you open and write an new issue. 

People which help you here have also limited time and resources. So by providing simple examples and test cases of your situation, you just enable others to help you more specific. Nice for everybody!

Unfortunately, there is a significant amount of very general or incomplete issue reports. These are not solvable, so to keep overview in the issue tracker, these are closed relatively quick.

**Comment by MGParisi** [comment]
As I stated in the linked documents, I had no additional information to give, and I gave up on the problem before I found a solution (or cause).  The information we do have is in the other link.  It seems to be related to the temporary storage of fonts on the server.  For me, it was because my test environment is Windows and this was NOT working.  The solutions seemed incomplete, and I could not reproduce it on an outside server so I abandoned my search.   Goodluck.
